content/pt-br/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -0,0 +1,400 @@
+```shell
+kubectl config --kubeconfig=config-demo set-context dev-frontend --cluster=development --namespace=frontend --user=developer
+kubectl config --kubeconfig=config-demo set-context dev-storage --cluster=development --namespace=storage --user=developer
+kubectl config --kubeconfig=config-demo set-context exp-test --cluster=test --namespace=default --user=experimenter
+```
+
+Abra seu arquivo `config-demo` para ver os detalhes adicionados. Como alternativa para abrir o arquivo `config-demo`, você pode usar o comando `config view`
+
+```shell
+kubectl config --kubeconfig=config-demo view
+```
+
+O resultado mostra os dois clusters, dois usuários e três contextos:
+
+```yaml
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: fake-ca-file
+    server: https://1.2.3.4
+  name: development
+- cluster:
+    insecure-skip-tls-verify: true
+    server: https://5.6.7.8
+  name: test
+contexts:
+- context:
+    cluster: development
+    namespace: frontend
+    user: developer
+  name: dev-frontend
+- context:
+    cluster: development
+    namespace: storage
+    user: developer
+  name: dev-storage
+- context:
+    cluster: test
+    namespace: default
+    user: experimenter
+  name: exp-test
+current-context: ""
+kind: Config
+preferences: {}
+users:
+- name: developer
+  user:
+    client-certificate: fake-cert-file
+    client-key: fake-key-file
+- name: experimenter
+  user:
+    # Documentation note (this comment is NOT part of the command output).
+    # Storing passwords in Kubernetes client config is risky.
+    # A better alternative would be to use a credential plugin
+    # and store the credentials separately.
+    # See https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins
+    password: some-password
+    username: exp
+```
+
+O `fake-ca-file`, o `fake-cert-file` e o `fake-key-file` acima são os espaços reservados
+para a localização dos arquivos de certificado. Você precisa alterá-los para a localização real
+dos arquivos de certificado em seu ambiente.
+
+Às vezes, você pode querer usar dados codificados em Base64 incorporados aqui, em vez de arquivos de certificado separados.
+Nesse caso, é necessário adicionar o sufixo `data` às chaves, por exemplo,
+`certificate-authority-data`, `client-certificate-data`, `client-key-data`.
+
+Cada contexto é uma tripla (cluster, usuário, namespace). Por exemplo, o contexto
+`dev-frontend` diz: "Use as credenciais do usuário `developer`
+para acessar o namespace `frontend` do cluster `development`".
+
+Define o contexto atual:
+
+```shell
+kubectl config --kubeconfig=config-demo use-context dev-frontend
+```
+
+Agora, sempre que você use um comando `kubectl`, a ação será aplicada ao cluster,
+e ao namespace listados no contexto `dev-frontend`. E o comando usará
+as credenciais do usuário listado no contexto `dev-frontend`.
+
+Para ver apenas as informações de configuração associadas ao
+o contexto atual, use o sinalizador `--minify`.
```

Comment:
What do you think of saying "opção" instead of "sinalizador"?

```suggestion
o contexto atual, use a opção `--minify`.
```

Reply:
Agreed. Sounds better.